autoreject/ransac.py

```diff
@@ -148,8 +152,9 @@ def _get_mappings(self, inst):
                                  for name in ch_subsets[idx]])
             mapping = np.zeros((n_channels, n_channels))
             if self.ch_type == 'meg':
-                mapping[:, pick_from] = _fast_map_meg_channels(inst, pick_from,
-                                                               pick_to)
+                # XXX Mainak here we need to similar things as in AR
+                mapping[:, pick_from] = _fast_map_meg_channels(
+                    inst.info.copy(), pick_from, pick_to)
```

Comment:
merci! sorry I couldn't make a lot of progress. Got interrupted many times. I'll try to continue after dinner if you don't beat me to it